package.json

```diff
@@ -43,7 +43,7 @@
     "throttleit": "^1.0.0",
     "unlimited": "^1.1.0",
     "webtorrent": "0.x",
-    "webtorrent-app": "feross/webtorrent-app",
+    "webtorrent-desktop": "git+ssh://git@github.com/feross/webtorrent-desktop.git",
```

Comment:
I think this is fine for the time being, but lmk if you think of a better way to handle getting the latest version.